.circleci/config.yml

```diff
@@ -18,26 +18,57 @@ jobs:
           paths:
             - providers
 
-  publish-release:
+  publish-staging-release:
     working_directory: ~/workspace
     docker:
       - image: cimg/base:stable
     steps:
       - checkout
       - gh/setup
+      - add_ssh_keys:
```

Comment:
is this needed? If so, don't we need an actual fingerprint?

Reply:
The recommended way in CircleCI was to have private key added in the project and public key in github repo then add this in steps.

Reply:
ah ok, then let's use a unique fingerprint for each environment

Reply:
unless ofc they use the same key